x/incentives/client/cli/tx.go

```diff
@@ -0,0 +1,172 @@
+package cli
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/version"
+	"github.com/cosmos/cosmos-sdk/x/gov/client/cli"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types"
+
+	"github.com/tharsis/evmos/x/incentives/types"
+)
+
+// NewRegisterIncentiveProposalCmd implements the command to submit a register
+//  incentive proposal
+func NewRegisterIncentiveProposalCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "register-incentive [contract-address] [allocation] [epochs]",
+		Args:    cobra.ExactArgs(3),
+		Short:   "Submit a proposal to register a contract incentive",
+		Long:    "Submit a proposal to register a contract incentive.",
+		Example: fmt.Sprintf("$ %s tx gov submit-proposal register-incentive <contract> --from=<key_or_address>", version.AppName),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			title, err := cmd.Flags().GetString(cli.FlagTitle)
+			if err != nil {
+				return err
+			}
+
+			description, err := cmd.Flags().GetString(cli.FlagDescription)
+			if err != nil {
+				return err
+			}
+
+			depositStr, err := cmd.Flags().GetString(cli.FlagDeposit)
+			if err != nil {
+				return err
+			}
+
+			deposit, err := sdk.ParseCoinsNormalized(depositStr)
+			if err != nil {
+				return err
+			}
+
+			if common.IsHexAddress(args[0]) {
+				return fmt.Errorf("invalid contract address: %s", args[0])
+			}
+
+			allocation, err := sdk.ParseDecCoins(args[1])
+			if err != nil {
+				return err
+			}
+
+			epochs, err := strconv.ParseUint(args[2], 10, 32)
+			if err != nil {
+				return err
+			}
+
+			contract := args[0]
+
+			from := clientCtx.GetFromAddress()
+			content := types.NewRegisterIncentiveProposal(title, description, contract, allocation, uint32(epochs))
+
+			msg, err := govtypes.NewMsgSubmitProposal(content, deposit, from)
+			if err != nil {
+				return err
+			}
+
+			if err := msg.ValidateBasic(); err != nil {
+				return err
+			}
+
+			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msg)
+		},
+	}
+
+	cmd.Flags().String(cli.FlagTitle, "", "title of proposal")
+	cmd.Flags().String(cli.FlagDescription, "", "description of proposal")
+	cmd.Flags().String(cli.FlagDeposit, "1aphoton", "deposit of proposal")
```

Comment:
we should change "aphoton" to "aevmos"
```suggestion
	cmd.Flags().String(cli.FlagDeposit, "1aevmos", "deposit of proposal")
```